src/MangaDexSharp/Models/Upload/UploadSessionCommit.cs

```diff
@@ -50,12 +50,12 @@ public class ChapterDraft
 	/// <summary>
 	/// Whether or not the chapter is hosted externally
 	/// </summary>
-	[JsonPropertyName("externalUrl")]
+	[JsonPropertyName("externalUrl"), JsonIgnore(Condition = JsonIgnoreCondition.Never)]
 	public string? ExternalUrl { get; set; }
 
 	/// <summary>
 	/// When the chapter is to be published
 	/// </summary>
-	[JsonPropertyName("publishAt")]
+	[JsonPropertyName("publishAt"), JsonIgnore(Condition = JsonIgnoreCondition.Never)]
 	public DateTime? PublishAt { get; set; }
```

Comment:
I think both properties needs to be ignored when null at request, based on [MangaDex API Swagger](https://api.mangadex.org/docs/swagger.html#/Upload/commit-upload-session), as i reported in #5. 

Probably all properties marked as required needs to be explicity written in json even if null; otherwise nonrequired props can't be present in json when null:

![image](https://github.com/calico-crusade/mangadex-sharp/assets/10831729/c668e5e2-f949-4c10-9800-485edf3fc2b0)

A weird behavior, if i can honestly say

Reply:
This should be fixed in the latest commit. Let me know if it works and I'll merge and deploy. Thanks.